pkg/tftp/config.go

```diff
@@ -5,12 +5,32 @@ import (
 	"time"
 )
 
+const (
+	defaultWriteTimeout = 5 * time.Second
+	defaultReadTimeout  = 5 * time.Second
+)
+
 type Config struct {
 	IP                     string
 	Port                   int
 	Retransmissions        int
 	MaxParallelConnections int
 	FileTransferTimeout    time.Duration
+	WriteTimeout           time.Duration
+	ReadTimeout            time.Duration
+	Metrics                metrics.Config
+}
+
+func (c Config) ReadTimeoutOrDefault() time.Duration {
+	if c.ReadTimeout == 0 {
```

Comment:
Wouldn't it make more sense to define a timeout of zero as no timeout at all? It would be unexpected for me if I set the timeout to zero explicitely and the program uses a timout of 5s in that case. :thinking:

Reply:
I think if you want to set it to 0 you just do not call the SetWrite/ReadDeadline()-Method. 
But espacially with the udp context I would not omit the timeout. If a packet gets lost the whole server could freeze.

Reply:
Ok, sorry, my mistake. Apparently I did not pay close attention ;) But it is still weird to me that the timeout is set to 5s, if it was previously set to 0s. Is there a reason you don't set the default directly via `viper.SetDefault()`

Reply:
agreed with @rageagainsthepc ... it is confusing to set viper to 0s and then default to 5s. 

viper should be 5s and default should be 5s.

Reply:
Changed the default timeout in Viper to 5s